lib/presentation/widget/upload_request_group/created/created_upload_request_group_widget.dart

```diff
@@ -0,0 +1,221 @@
+import 'package:linshare_flutter_app/presentation/widget/upload_request_group/created/created_upload_request_group_viewmodel.dart';
+import 'package:linshare_flutter_app/presentation/util/extensions/datetime_extension.dart';
+import 'package:redux/redux.dart';
+import 'package:linshare_flutter_app/presentation/redux/states/ui_state.dart';
+import 'package:linshare_flutter_app/presentation/view/order_by/order_by_button.dart';
+
+class CreatedUploadRequestGroupWidget extends StatefulWidget {
+  CreatedUploadRequestGroupWidget({Key? key}) : super(key: key);
+
+  @override
+  _CreatedUploadRequestGroupWidgetState createState() => _CreatedUploadRequestGroupWidgetState();
+}
+
+class _CreatedUploadRequestGroupWidgetState extends State<CreatedUploadRequestGroupWidget> with AutomaticKeepAliveClientMixin {
+  final _model = getIt<CreatedUploadRequestGroupViewModel>();
+  final imagePath = getIt<AppImagePaths>();
+
+  @override
+  void initState() {
+    Future.delayed(Duration.zero, () {
+      _model.initState();
+    });
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _model.onDisposed();
+    super.dispose();
+  }
+
+  @override
+  bool get wantKeepAlive => true;
+
+  @override
+  Widget build(BuildContext context) {
+		super.build(context);
+    return StoreConnector<AppState, bool>(
+        converter: (store) => store.state.uiState.isInSearchState(),
+        builder: (context, isInSearchState) =>
+					isInSearchState 
+						? _buildSearchViewWidget(context) 
+						: _buildDefaultViewWidget());
+  }
+
+  Widget _buildDefaultViewWidget() {
+    return StoreConnector<AppState, CreatedUploadRequestGroupState>(
+      converter: (store) => store.state.createdUploadRequestGroupState,
+      builder: (_, state) => Column(children: [
+				_buildMenuSorterPending(),
+				Expanded(
+						child: RefreshIndicator(
+								onRefresh: () async => _model.getUploadRequestCreatedStatus(),
+								child: _buildListData(state.uploadRequestsCreatedList)))
+			])
+    );
+  }
+
+  Widget _buildMenuSorterPending() {
+    return StoreConnector<AppState, AppState>(
+      converter: (Store<AppState> store) => store.state,
+      builder: (context, appState) {
+        return !appState.uiState.isInSearchState()
+            ? OrderByButtonBuilder(context, appState.createdUploadRequestGroupState.pendingSorter)
+                .onOpenOrderMenuAction(
+                    (currentSorter) => _model.openPopupMenuSorterPending(context, currentSorter))
+                .build()
+            : SizedBox.shrink();
+      },
+    );
+  }
+
+  Widget _buildListData(List<SelectableElement<UploadRequestGroup>> uploadRequestList) {
+    if (uploadRequestList.isEmpty) {
+      return _buildCreateUploadRequestsHere(context);
+    } else {
+      return ListView.builder(
+          itemCount: uploadRequestList.length, 
+					itemBuilder: (context, index) => 
+						_buildPendingUploadTile(uploadRequestList[index]));
```

Comment:
indent

Reply:
In my local machine

<img width="730" alt="Screen Shot 2021-08-23 at 11 36 48" src="https://user-images.githubusercontent.com/29337364/130390627-03ba6ad5-8e7e-4244-869e-941e2ba5b8d8.png">